exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,23 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+var is80orMore = "A";
+var is60to80 = "B"
+var is50to60 = "C"
+var isLessThan50 = "F"
+
+
+function calculateGrade(mark) {
+  if (mark >= 80){                     //checks if mark is 80 or more
+    return is80orMore         
+  } else if (mark >= 60){              // checks if mark is 60 or more
+    return is60to80
+  } else if ( mark === 60 || mark >= 50){  // checks if mark is 50 or more
```

Comment:
Because on line 20, you have (mark >= 60), if it is 60, it will never be "C". Also, can you convert this to a "switch" statement? I think you will enjoy learning about it!